frontend/src/constants.js

```diff
@@ -341,4 +341,27 @@ export default Object.freeze({
       label: "Informations sur le site",
     },
   },
+  DiagnosticImportLevels: [
+    {
+      key: "NONE",
+      title: "Importer des cantines sans diagnostic",
+      label: "Sans diagnostic",
+      help: "Vous voulez importer des cantines sans diagnostic",
```

Comment:
Le titre et le help se ressemblent beaucoup, ça n'apporte pas d'info supplémentaire comme pour les autres. Est-ce que ça t'irait ?

```suggestion
      help: "Vous voulez importer des cantines sans données d'approvisionnement",
```